src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VirtualProject.cs

```diff
@@ -21,6 +24,46 @@ internal static class VirtualCSharpFileBasedProgramProject
     internal static string GetVirtualProjectPath(string documentFilePath)
         => Path.ChangeExtension(documentFilePath, ".csproj");
 
+#region TODO: Copy-pasted from dotnet run-api. Delete when run-api is adopted.
+    private static class Sha256Hasher
+    {
+        /// <summary>
+        /// The hashed mac address needs to be the same hashed value as produced by the other distinct sources given the same input. (e.g. VsCode)
+        /// </summary>
```

Comment:
I'm guessing this comment got carried along and can be deleted since it doesn't really make sense.